docs/docs-content/release-notes/known-issues.md

```diff
@@ -17,7 +17,7 @@ The following table lists all known issues that are currently active and affecti
-| Self-hosted Palette and Palette VerteX [Enterprise Clusters (ECs)](../automation/palette-cli/commands/ec.md) and [Private Cloud Gateway (PCG) clusters](../automation/palette-cli/commands/pcg.md) deployed with Calico version 3.28.2, 3.29.0, or 3.29.1 are experiencing memory leaks due to an [upstream issue](https://github.com/projectcalico/calico/pull/9612) with Calico, which is caused by failing to close netlink handles.                                                                                                                                                   | [Create a new profile version](../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md) using Calico version 3.28.0 or 3.28.1 and [update your cluster](../clusters/cluster-management/cluster-updates.md#update-a-cluster).                                                                                                                                                                                                                                                                                                                                                            | January 27, 2025   | Clusters, Packs              |
+| Palette [workload clusters](../glossary-all.md#workload-cluster) deployed with Calico version 3.28.2, 3.29.0, or 3.29.1 are experiencing memory leaks due to an [upstream issue](https://github.com/projectcalico/calico/pull/9612) with Calico, which is caused by failing to close netlink handles.                                                                                                                                                                                                                                                                                     | [Create a new profile version](../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md) using Calico version 3.28.0 or 3.28.1 and [update your cluster](../clusters/cluster-management/cluster-updates.md#update-a-cluster).                                                                                                                                                                                                                                                                                                                                                            | January 27, 2025   | Clusters, Packs              |
```

Comment:
This issue is applicable for both **E**nterprise **C**lusters and **W**orkload **C**lusters.